src/data/actions/courseInfo.js

```diff
@@ -134,7 +134,8 @@ function createCourse(courseData, courseRunData) {
   };
 }
 
-function editCourse(courseData, courseRunData) {
+function editCourse(courseData, courseRunData, submittingRunForReview = false) {
+  const submitReview = submittingRunForReview;
```

Comment:
Why not just use the input parameter?

Reply:
For the variable to be in the scope of the callback it has to be a Constant.  So I convert it to a constant here.